src/Storage/ABS/SourceFile.php

```diff
@@ -73,10 +74,41 @@ public function getManifestEntries(): array
             $SASConnectionString
         );
 
-        $getResult = $blobClient->getBlob($this->container, $this->filePath);
-        $manifest = \GuzzleHttp\json_decode(stream_get_contents($getResult->getContentStream()), true);
-        return array_map(function ($entry) {
-            return $entry['url'];
+        if (!$this->isSliced) {
+            // this is temporary solution copy into is not failing when blob not exists
+            try {
+                $blobClient->getBlob($this->container, $this->filePath);
+            } catch (ServiceException $e) {
+                throw new Exception('Load error: ' . $e->getErrorText(), Exception::MANDATORY_FILE_NOT_FOUND, $e);
+            }
+
+            return [$this->getContainerUrl($protocol) . $this->filePath];
+        }
+
+        try {
+            $manifestBlob = $blobClient->getBlob($this->container, $this->filePath);
+        } catch (ServiceException $e) {
+            throw new Exception('Load error: manifest file was not found.', Exception::MANDATORY_FILE_NOT_FOUND, $e);
+        }
+
+        $manifest = \GuzzleHttp\json_decode(stream_get_contents($manifestBlob->getContentStream()), true);
+        return array_map(function ($entry) use ($protocol, $blobClient) {
+            // this is temporary solution copy into is not failing when blob not exists
```

Comment:
Jaký bude finální? Asi bysme se  mohli na ty manifesty vyprdnou a loadovat vše co je v tom kontjneru....

Reply:
Bylo by to nejlepší, ale musíme se zbavit manifestu. Buď ho smazat, nevytvářet  nebo možná nastavit jako hidden. Ani popravdě nevím kde sa ten manifest pro import vytvoří. Po exportu ho vytvoří connection, ale kde sa udělá pro import nevím.

Reply:
Jo jasně on tam udělá paseku ten manifest protože ho to taky chce importnout a padne to na něm. To odstranění a nepoužívání by bylo potřeba promyslet a rozdělit do více kroků - Takže samostatný epic/story. Možná třeba říct že Synapse s ním nebude pracovat, muselo by ho to ale umět přeskočit.

Ten manifest musí uploadnout člověk sám když vytváří sliced soubor. A u nás to za tebe řeší SAPI klient při uploadu sliced souboru:
- S3 https://github.com/keboola/storage-api-php-client/blob/ed6ab7569cdb4d060c2d4ba2bbfa6889c27f08a2/src/Keboola/StorageApi/Client.php#L1642
- ABS https://github.com/keboola/storage-api-php-client/blob/ed6ab7569cdb4d060c2d4ba2bbfa6889c27f08a2/src/Keboola/StorageApi/Client.php#L1586

Reply:
Ještě sem přemýšlel a v dokumentaci je:
```
COPY ignores hidden folders and doesn't return files that begin with an underline (_) or a period (.) unless explicitly specified in the path. This behavior is the same even when specifying a path with a wildcard.
```
takže kdybysme na začátku importu ten manifest přejmenovali takže na začátek dáme `_` nebo `.` a na konci importu to vrátíme zpátky tak sa mu "relativně" elegantně vyhneme.

Reply:
Ono ale teoreticky k tomu stejnému souboru během toho importu může přistupovat i něco jiného, i když si myslím že reálně ta situace nikde nenastává.

Reply:
Asi bych do téhle varianty teda radši spíš nešel.